docs/public-networks/how-to/pre-merge-history-expiry.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Prune pre-merge history
+sidebar_position: 9
+description: Reduce the size of your database by removing pre-merge blocks from your blockchain
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Node operators using [Snap sync](../concepts/node-sync.md#snap-synchronization) can significantly reduce
+disk usage by removing [pre-merge](https://ethereum.org/en/roadmap/merge/) Proof of Work (PoW)
+block data from the local database.
+
+Besu can prune all pre-merge blocks and associated transaction receipts, leaving only headers and
```

Comment:
```suggestion
Besu can prune all pre-merge block bodies and associated transaction receipts, leaving only headers and
```

Reply:
for context, block bodies are made up of transactions, ommer headers and withdrawals (though no withdrawals pre-merge). The transaction data is by far the biggest part.